cassandra-3/src/main/java/io/debezium/connector/cassandra/Cassandra3CommitLogReadHandlerImpl.java

```diff
@@ -665,4 +665,8 @@ private List<Object> getPartitionKeys(PartitionUpdate pu) {
         return values;
     }
 
+    private int getPartitionQueueIndex(PartitionUpdate partitionUpdate) {
```

Comment:
Can this method be extracted into a utility class in core module and re-used in all three modules?